.setup/INSTALL_SUBMITTY_HELPER.sh

```diff
@@ -706,31 +706,31 @@ original_autograding_workers=/var/local/submitty/autograding_TODO/autograding_wo
 if [ -f $original_autograding_workers ]; then
     temp_autograding_workers=`mktemp`
```

Comment:
Consider using `$(...)` notation?

```suggestion
    temp_autograding_workers=$(mktemp)
```